tests/unit_tests/test_wave_02.py

```diff
@@ -2,30 +2,36 @@
 from swap_meet.vendor import Vendor
 from swap_meet.item import Item
 
-@pytest.mark.skip
 def test_items_have_default_uuid_length_id():
     item = Item()
     assert isinstance(item.id, int)
     assert len(str(item.id)) >= 32
 
-@pytest.mark.skip
 def test_item_instances_have_different_default_ids():
     item_a = Item()
     item_b = Item()
     assert item_a.id != item_b.id
 
-@pytest.mark.skip
 def test_items_use_custom_id_if_passed():
     item = Item(id=12345)
     assert isinstance(item.id, int)
     assert item.id == 12345
 
-@pytest.mark.skip
+#my tests:
+def test_items_if_non_integer_id_raises_an_error():
+    test_id = "y"
+    with pytest.raises(TypeError):
+        Item(test_id) 
+
+def test_items_if_non_integer_condition_raises_an_error():
```

Comment:
Excellent job adding new test cases for the edge cases you're handling! Great job taking on the optional enhancements!